frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,95 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-user-preferences"
+              And user is at Add page
+
+
+        @smoke @to-do
+        Scenario: Access user preference from the masthead: UP-01-TC01
+             When user clicks on user dropdown
+             Then user will see User Preference
+
+
+        @regression @to-do
+        Scenario: Setting Developer preference for perspective: UP-01-TC02
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Perspective dropdown on User Preferences page
+              And user selects "Developer" option
+             Then user can see "Developer" option selected in the perspective switcher
+
+
+        @regression @to-do
+        Scenario: Setting a preference for a project: UP-01-TC03
+            Given user has created project "test-preference1"
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Project dropdown on User Preferences page
+              And user searches project "test-preference1"
+              And user selects the project "test-preference1" from the dropdown
+              And user clicks on Add from navigation menu
```

Comment:
```suggestion
              And user clicks on Add present in navigation menu
```

Reply:
Clicking on `Add` from navigation menu will not reflect any change since the link associated with this menu option uses the active namespace.
To notice a change after setting the preference for a Project, the user would have to load the console without any url params for namespace.
cc @jerolimov

Reply:
It is true that the user needs to reload the console to get the selected perspective. But I think the selected project should be also visible if the user switches from the admin perspective to the developer perspective, or?